src/table_builder_test.cc

```diff
@@ -318,6 +344,121 @@ TEST_F(TableBuilderTest, NumEntries) {
   ASSERT_OK(table_builder->Finish());
 }
 
+TEST_F(TableBuilderTest, TargeSize) {
+  cf_options_.blob_file_target_size = kTargetBlobFileSize;
+  table_factory_.reset(new TitanTableFactory(
+      db_options_, cf_options_, blob_manager_, &mutex_, vset_.get(), nullptr));
+  std::unique_ptr<WritableFileWriter> base_file;
+  NewBaseFileWriter(&base_file);
+  std::unique_ptr<TableBuilder> table_builder;
+  NewTableBuilder(base_file.get(), &table_builder);
+  const int n = 255;
+  for (unsigned char i = 0; i < n; i++) {
+    std::string key(1, i);
+    InternalKey ikey(key, 1, kTypeValue);
+    std::string value(kMinBlobSize, i);
+    table_builder->Add(ikey.Encode(), value);
+  }
+  ASSERT_OK(table_builder->Finish());
+  uint64_t file_size = 0;
+  ASSERT_OK(env_->GetFileSize(blob_name_, &file_size));
```

Comment:
I look at the code and seems the test table builder don't write to file with name as `blob_name_`. Am I missing something?

Reply:
> I look at the code and seems the test table builder don't write to file with name as `blob_name_`. Am I missing something?

Since total size to write is larger than blob_file_target_size, it will write to multiple blob file with size about blob_file_target_size, blob_name_ is the first one.

Reply:
@yiwu-arbug I modified the logic